lib/models/BucketInfo.js

```diff
@@ -521,6 +535,20 @@ class BucketInfo {
         return this._versioningConfiguration &&
             this._versioningConfiguration.Status === 'Enabled';
     }
+    /**
+     * Get unique id of bucket.
+     * @return {string} - unique id
+     */
+    getUid() {
+        return this._uid;
+    }
+    /**
+     * Set unique id for bucket.
+     * @return {BucketInfo} - bucket info instance
+     */
+    setUid() {
```

Comment:
This is not technically a setter, so to avoid confusion I would call it e.g. `genUid()`.

Reply:
or `generateUid` which is more self-explanatory

Reply:
or `createUid()` which also carries the notion of setting it